cfgov/jinja2/v1/_includes/molecules/link-blob.html

```diff
@@ -0,0 +1,14 @@
+<div class="o-link-blob">
```

Comment:
Can you change this to `m-half-width-link-blob`? That's what we have in the LESS.